src/List/ListItemText.js

```diff
@@ -14,6 +14,9 @@ export const styleSheet = createStyleSheet('ListItemText', () => {
         paddingLeft: 0,
       },
     },
+    inset: {
+      paddingLeft: '56px !important',
```

Comment:
We need to do something different. Let me think about it... I really want to avoid doing this. Really really.